recipes-core/images/core-image-minimal.bbappend

```diff
@@ -1,16 +1,14 @@
-IMAGE_FEATURES_append = " ssh-server-openssh"
+IMAGE_FEATURES:append = " ssh-server-openssh"
 
-IMAGE_INSTALL_append = " \
-   packagegroup-rte-core \
-   packagegroup-rte-utils \
-   packagegroup-rte-imx \
-   packagegroup-rte-stm \
-   packagegroup-rte-coreboot \
-   locale-base-en-us \
-   "
-
-# set root password
-inherit extrausers
+IMAGE_INSTALL:append = " \
+    packagegroup-rte-core \
+    packagegroup-rte-utils \
+    packagegroup-rte-imx \
+    packagegroup-rte-stm \
+    packagegroup-rte-coreboot \
+    locale-base-en-us \
+    "
 
+INHERIT += "extrausers"
```

Comment:
@PLangowski 

> [INHERIT](https://docs.yoctoproject.org/ref-manual/variables.html#term-INHERIT)[](https://docs.yoctoproject.org/ref-manual/variables.html#term-INHERIT)
>
>    Causes the named class or classes to be inherited globally. Anonymous functions in the class or classes are not executed for the base configuration and in each individual recipe. **The OpenEmbedded build system ignores changes to [INHERIT](https://docs.yoctoproject.org/ref-manual/variables.html#term-INHERIT) in individual recipes**

I quoted most important part in **bold**. You should probably move this to distro conf file or use

```
inherit extrausers
```

Reply:
26663e11e20251cd683c4b0770d8e9c00541114a